ractor.rb

```diff
@@ -308,9 +314,10 @@ def self.count
   #
   #    puts "received #{obj.inspect} from #{r.inspect}"
   #    # Prints: received "from 1" from #<Ractor:#2 test.rb:1 running>
+  #    # But could just as well print "from r2" here, either of them could be the first.
```

Comment:
```suggestion
  #    # This could print "from r2" here, either prints could be first.
```

I'm always weary of excessive words, especially in a comment. Let's try to be as concise as possible for all non-native readers too.

Reply:
I was just trying to point out, it's not deterministic. If someone tries the example, it might not print that.

Reply:
Gotcha 👍